common/src/main/java/com/wynntils/overlays/minimap/MinimapOverlay.java

```diff
@@ -93,6 +94,11 @@ public MinimapOverlay() {
                 new OverlaySize(DEFAULT_SIZE, DEFAULT_SIZE));
     }
 
+    public void scale(double scaler) {
+        scale.setValue((float) Math.clamp(0.1, 3, scale.get() * scaler));
```

Comment:
extract 0.1 and 3 into static finals at the top of the class

Reply:
These are the same as in abstract map screen. Should I use them instead?